iotcore/src/test/java/com/google/android/things/iotcore/ConnectionParamsTest.java

```diff
@@ -245,4 +245,12 @@ public void testGetConfigurationTopic() {
         String expectedConfigurationTopic = "/devices/" + DEVICE_ID + "/config";
         assertThat(connectionParams.getConfigurationTopic()).isEqualTo(expectedConfigurationTopic);
     }
+
+    @Test
+    public void testGetCommandsTopicPrefix() {
```

Comment:
function name would be "testGetCommandsTopic" as we agreed.

Reply:
my bad: sloppy...